specification/subscription/resource-manager/readme.md

```diff
@@ -24,11 +24,36 @@ To see additional help and options, run:
 ### Basic Information 
 These are the global settings for the Subscription API.
 
+``` yaml
+openapi-type: arm
+tag: package-2018-03-01-preview
+```
+
 ``` yaml
 openapi-type: arm
 tag: package-2017-11-preview
```

Comment:
Remove this section - the global reference should point to the latest version always.